api-report/container-runtime.api.md

```diff
@@ -511,8 +511,6 @@ export interface ISummarizerRuntime extends IConnectableRuntime {
 export interface ISummarizingWarning extends ContainerWarning {
     // (undocumented)
     readonly errorType: "summarizingError";
-    // (undocumented)
-    readonly logged: boolean;
```

Comment:
Is `ISummarizingWarning` intended to be used exclusively internally?  If so you only need to worry about the runtime compatibility (so probably would still need to make this optional).  Otherwise yes this is a major breaking change, but it would be fine if you were to make it optional.